tftrt/examples/image-classification/image_classification.py

```diff
@@ -30,6 +30,10 @@
 import official.resnet.imagenet_main
 from preprocessing import inception_preprocessing, vgg_preprocessing
 
+tf.enable_eager_execution()
```

Comment:
why do we have eager execution here?

Reply:
You are right, we don't need to use it. I will change that.